sass/directives/_carousels.scss

```diff
@@ -123,12 +123,14 @@
   border-top: $base-border;
   border-color: $grey-lighter;
   background-color: $brand-new-grey-lightest;
-  padding-top: $base-margin - 0.25rem;
-  padding-bottom: $base-margin - 0.25rem;
-  padding-left: 1em;
+  // padding-top: $base-margin - 0.25rem;
+  // padding-bottom: $base-margin - 0.25rem;
```

Comment:
@ElliottAYoung yep, I was just going a quick test, next time I will just delete them, since i can always just revert. Sorry about that.